fhir-config/src/main/java/com/ibm/fhir/config/FHIRRequestContext.java

```diff
@@ -302,4 +310,70 @@ public boolean isReturnPreferenceDefault() {
     public void setReturnPreferenceDefault(boolean returnPreferenceIsDefault) {
         this.returnPreferenceDefault = returnPreferenceIsDefault;
     }
-}
+
+    /**
+     * Get a handle to track a new instance of a call. Metric names must not include the
+     * forward slash '/' character because it is used as a path separator
+     * @param name
+     * @return
+     */
+    public MetricHandle getMetricHandle(String name) {
+        if (name.contains("/")) {
+            throw new IllegalArgumentException("Metric names MUST NOT include the metric path separator '/'");
+        }
+
+        if (log.isLoggable(Level.FINEST)) {
+            if (currentMetricHandle != null) {
+                log.finest("getMetricHandle: '" + name + "', currentMetric is '" + currentMetricHandle.getPath() + "'");
+            } else {
+                log.finest("getMetricHandle: '" + name + "', currentMetric is '/'");
+            }
+        }
+
+
+        final String parentPath = currentMetricHandle == null ? "/" : currentMetricHandle.getPath();
```

Comment:
very minor style thing:  I've started putting the ternary predicate in parens.  I think I got that from John.  I think its very slightly more readable.  See if you agree:
```suggestion
        final String parentPath = (currentMetricHandle == null) ? "/" : currentMetricHandle.getPath();
```

Reply:
Yeah, a lot of the time I baulk at over-use of parens but in this case I think it's justified. It's similar to mandating braces after `if` even though they're not required for single statements. Added.